phpBB/language/en/ucp.php

```diff
@@ -142,7 +136,7 @@
 
 	'CONFIRM_PASSWORD'			=> 'Confirm password',
 	'CONFIRM_PASSWORD_EXPLAIN'	=> 'You only need to confirm your password if you changed it above.',
-	'COPPA_BIRTHDAY'			=> 'To continue with the registration procedure please tell us when you were born.',
+	'COPPA_BIRTHDAY'			=> '<p>To continue with the registration procedure please tell us when you were born.</p>',
```

Comment:
Why aren't these P tags in the template?

Reply:
Changed.